code/src/components/Name.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+import React from 'react';
+
+export const Name = ({ name, setName }) => {
+    const handleNameChange = (event) => {
+        const upCaseUserName = event.target.value.charAt(0).toUpperCase() + event.target.value.slice(1);
+        setName(upCaseUserName);
+};
```

Comment:
I found it impressive that it was taken into consideration that the user might not capitalize their name. Nice addition.